bin/plugin/commands/performance.js

```diff
@@ -220,21 +222,32 @@ async function runPerformanceTests( branches, options ) {
 	log( '>> Starting the WordPress environment' );
 	await runShellScript( 'npm run wp-env start', environmentDirectory );
 
-	/** @type {Record<string, WPFormattedPerformanceResults>} */
+	const testSuites = [ 'post-editor', 'site-editor' ];
```

Comment:
Should this be an argument instead of running all tests with default to post-editor? or do you think we should be running all tests everytime?

Reply:
This is in the function body of `runPerformanceTests`, which is the top-level export of this file (invoked by `bin/plugin/cli.js`). If we want to make this an argument, we'd basically have to make it one that's passed by the CLI.

I'm leaning towards always running both perf suites. Their main use case is CI (the GH Action), as this is the most reliable guard against perf regressions, so it should IMO be easy to monitor that. Furthermore, the Site Editor test only covers loading time, which is currently pretty short. We can revisit if we add more coverage and testing times grow too much.